financial-services-accelerator/accelerators/fs-apim/repository/conf/configure.properties

```diff
@@ -40,6 +40,5 @@ DB_DRIVER=com.mysql.jdbc.Driver
 DB_APIMGT=fs_apimgtdb
 DB_USER_STORE=fs_userdb
 DB_AM_CONFIG=fs_am_configdb
-DB_GOV=fs_govdb
```

Comment:
dont we need this now?

Reply:
noWith IS. 7.1 we cant share the DB s so we are using config db as gov db also